blueprints/helper/files/app/helpers/__name__.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
-export function <%= camelizedModuleName %>(input) {
-  return input;
+export function <%= camelizedModuleName %>(params, hash, options, env) {
```

Comment:
You should likely put the args in comments (otherwise the JSHint tests will blow up after generating):

```javascript
export function <%= camelizedModuleName %>(/* params, hash, options, env */) {
}
```

And I think that the generate tests will also need to be updated.